CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKCommandBuffer.java

```diff
@@ -146,8 +180,294 @@ public void FinishRecord(){
             throw new AssertionError("Failed to finish record command buffer: ");
         }
     }
+	
+	public void beginRenderPassCmd(long renderPass, long frameBuffer,
```

Comment:
This kinda violates our naming conventions of CamelCase and avoiding contractions (cmd)

Reply:
Yep it is WIP, there are still some more functions to bring in  then I'll do a pass over it all